src/hooks/useScanner.js

```diff
@@ -136,13 +137,21 @@ export default function useScanner(enabled) {
       analytics.track('Scanned WalletConnect QR code');
       await checkPushNotificationPermissions();
 
-      try {
-        await walletConnectOnSessionRequest(qrCodeData, () => {
-          setTimeout(enableScanning, 2000);
-        });
-      } catch (e) {
-        logger.log('walletConnectOnSessionRequest exception', e);
-        setTimeout(enableScanning, 2000);
+      const { query, pathname } = new URL(qrCodeData);
+      if (qrCodeData && query) {
+        const [, version] = pathname.split('@');
+        if (version === '2') {
```

Comment:
this "2" is in a few places, should we move it to an enum WC_VERSION_NUMBER instead?